packages/flutter/lib/src/foundation/debug.dart

```diff
@@ -132,3 +134,31 @@ String? activeDevToolsServerAddress;
 
 /// The uri for the connected vm service protocol.
 String? connectedVmServiceUri;
+
+/// If memory allocations are enabled, dispatch object creation.
+///
+/// This method is not member of FlutterMemoryAllocations, because
+/// [FlutterMemoryAllocations] should not increase size of the Flutter application
+/// if memory allocations are disabled.
+///
+/// Returns true to make it easier to be wrapped into `assert`.
```

Comment:
Maybe specify: Should this only be called from within an assert? When would you call it outside of an assert?